cmd/dist/dist.go

```diff
@@ -33,17 +32,13 @@ func getTargets() ([]dist.Target, error) {
 	// Since only we can provide packages to Synology for
 	// distribution, we default to building the "sideload" variant of
 	// packages that we distribute on pkgs.tailscale.com.
+	synologyPackageCenter := slices.Contains(args, "--synology-package-center")
```

Comment:
this seems like a brittle way to check for a flag, and would miss using a single leading dash, `-synology-package-center`.  Can we pass an actual struct here, or a flagset?  Just glancing at this diff, I'm not sure if or where else a similar getTargets method is used.

Reply:
I was hoping to pass something more structured, but the place where this actually gets called has the args in slice form.
```
Exec: func(ctx context.Context, args []string) error {
        targets, err := getTargets(args)
        ...
```
I'll play with it a little more. `getTargets` is pretty specific to this builder, so I think we have some room here. But agreed it is unideal. `--synology-package-center=true` does still evaluate to `--synology-package-center` in the slice though, so it it's a little less brittle than it looks.